client/components/cart-songs-view.js

```diff
@@ -0,0 +1,94 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import {fetchCart, setLocalStorageItem} from '../store/cart'
+import {
+  Button,
+  Card,
+  Container,
+  Media,
+  Row,
+  Col,
+  Image,
+  ListGroup
+} from 'react-bootstrap'
+
+class Cart_Songs_View extends React.Component {
+  constructor() {
+    super()
+  }
+
+  async componentDidMount() {
+    await this.props.getCart()
+    await this.props.setLocalStorage(this.props.currentCart)
+  }
+
+  render() {
+    const cart = this.props.currentCart
+    const cartSongs = cart.songs
+
+
+    return (
+      <Container fluid>
+        <div>
+          <section>
+            <h3>Songs:</h3>
+            <ListGroup className="list-unstyled">
+              {cartSongs ? (
+                <div>
+                  {cartSongs.map(song => (
+                    <Media as="li" key={song.id}>
+                      <img
+                        width={150}
+                        height={150}
+                        className="align-self-center mr-3"
+                        src={song.imageUrl}
+                        className="shadow-lg mb-5 bg-white"
+                      />
+                      <Media.Body>
+                        <h4 className="cart-song-details-name">
+                          Song: "{song.name}"
+                        </h4>
+                        <p className="cart-song-details">
+                          Artist: {song.artist}
+                        </p>
+                        <p className="cart-song-details">Album: {song.album}</p>
+                      </Media.Body>
+                      <Button
+                        className="cart-song-delete-button"
+                        variant="outline-secondary"
+                        size="sm"
+                      >
+                        <img
+                          width={25}
+                          height={25}
+                          src="https://cdn1.iconfinder.com/data/icons/basic-ui-elements-12/512/Basic_Icons-Mini-Source-04-512.png"
+                        />Remove Song
+                      </Button>
+                    </Media>
+                  ))}
+                </div>
+              ) : (
+                'Empty Cart'
```

Comment:
I don't think we need this here, because my main cart component is checking that at a higher level. so, if we don't have any songs in the cart, we wouldn't even get to render the list, main cart component will just render empty cart page with a button

Reply:
Removed!